data/json/recipes/food/veggy.json

```diff
@@ -0,0 +1,222 @@
+    "components": [
+      [ [ "water", 1 ], [ "water_clean", 1 ], [ "aluminum_foil", 7 ] ],
+      [ [ "mushroom", 1 ] ]
+    ]
+  },
+  {
+    "result": "dahlia_baked",
+    "type": "recipe",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_VEGGI",
+    "skill_used": "cooking",
+    "difficulty": 1,
+    "time": 12000,
+    "reversible": false,
+    "autolearn": false,
+    "book_learn": [ [ "pocket_survival", 1 ], [ "survival_book", 1 ], [ "textbook_survival", 1 ], [ "manual_survival", 1 ], [ "survival_book", 1 ] ],
+    "qualities": [ { "id": "COOK", "level": 1 } ],
+    "tools": [ [ [ "hotplate", 5 ], [ "char_smoker", 5 ], [ "toolset", 5 ], [ "fire", -1 ] ] ],
+    "components": [
+      [ [ "water", 1 ], [ "water_clean", 1 ], [ "aluminum_foil", 5 ] ],
+      [ [ "dahlia_root", 1 ] ]
+    ]
+  },
+  {
+    "result": "potato_baked",
+    "type": "recipe",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_VEGGI",
+    "skill_used": "cooking",
+    "difficulty": 1,
+    "time": 15000,
+    "autolearn": true,
+    "qualities": [ { "id": "COOK", "level": 1 } ],
+    "tools": [ [ [ "hotplate", 3 ], [ "char_smoker", 3 ], [ "toolset", 3 ], [ "fire", -1 ] ] ],
+    "components": [
+      [ [ "water", 1 ], [ "water_clean", 1 ], [ "aluminum_foil", 3 ] ],
+      [ [ "potato_raw", 1 ], [ "potato_irradiated", 1 ] ]
+    ]
+  },
+  {
+    "result": "veggy_cooked",
+    "type": "recipe",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_VEGGI",
+    "skill_used": "cooking",
+    "difficulty": 0,
+    "time": 4000,
+    "reversible": false,
+    "autolearn": true,
+    "qualities": [ { "id": "COOK", "level": 1 } ],
+    "tools": [ [ [ "hotplate", 5 ], [ "char_smoker", 5 ], [ "toolset", 5 ], [ "fire", -1 ] ] ],
+    "components": [
+      [ [ "water", 1 ], [ "water_clean", 1 ], [ "aluminum_foil", 5 ] ],
+      [ [ "veggy", 1 ] ]
+    ]
+  },
+  {
+    "result": "veggy_wild_cooked",
+    "type": "recipe",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_VEGGI",
+    "skill_used": "cooking",
+    "difficulty": 0,
+    "time": 4000,
+    "autolearn": true,
+    "qualities": [ { "id": "COOK", "level": 2 } ],
+    "tools": [ [ [ "hotplate", 5 ], [ "char_smoker", 5 ], [ "toolset", 5 ], [ "fire", -1 ] ] ],
+    "components": [
+      [ [ "water", 1 ], [ "water_clean", 1 ], [ "aluminum_foil", 5 ] ],
+      [ [ "veggy_wild", 1 ] ]
+    ]
+  },
+  {
+    "result": "cooked_cattail_stalk",
+    "type": "recipe",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_VEGGI",
+    "skill_used": "cooking",
+    "difficulty": 1,
+    "skills_required": [ "survival", 1 ],
+    "time": 18000,
+    "autolearn": true,
+    "byproducts": "plant_fibre",
+    "qualities": [
+      { "id": "CUT", "level": 1 },
+      { "id": "COOK", "level": 1 }
+    ],
+    "tools": [ [ [ "hotplate", 5 ], [ "char_smoker", 5 ], [ "toolset", 5 ], [ "fire", -1 ] ] ],
+    "components": [
+      [ [ "water", 1 ], [ "water_clean", 1 ], [ "aluminum_foil", 5 ] ],
```

Comment:
Stalk originally didn't take water, suggesting it can be cooked over fire. If it is to take water/foil now, it should require COOK 2.